### PR TITLE
Update ibc-go version to v1.0.0-beta1

### PR DESCRIPTION
**Pull request by bluele**

### CODE REVIEW

File: app/baseapp.go [hunk 1]
```diff
@@ -26,20 +28,20 @@ type Application interface {
...
-	queryRouter      sdk.QueryRouter   // router for redirecting query calls
-	grpcQueryRouter  *GRPCQueryRouter  // router for redirecting gRPC query calls
-	msgServiceRouter *MsgServiceRouter // router for redirecting Msg service messages
-	txDecoder        sdk.TxDecoder     // unmarshal []byte into sdk.Tx
+	name             string                    // application name from abci.Info
+	db               dbm.DB                    // common DB backend
+	cms              *store.Store              // Main (uncached) state
+	router           sdk.Router                // handle any kind of message
+	queryRouter      sdk.QueryRouter           // router for redirecting query calls
+	grpcQueryRouter  *GRPCQueryRouter          // router for redirecting gRPC query calls
```
Comment (3100) [inline app/baseapp.go:42]: `GRPCQueryRouter` (and `app/grpcrouter.go`) looks the same as cosmos/cosmos-sdk at a quick glance, is there any reason why we are using our own definition?

File: app/baseapp.go [hunk 2]
```diff
@@ -61,10 +63,10 @@ func NewBaseApp(
 		name:             name,
 		db:               db,
 		cms:              store.NewStore(db),
-		router:           NewRouter(),
-		queryRouter:      NewQueryRouter(),
+		router:           baseapp.NewRouter(),
+		queryRouter:      baseapp.NewQueryRouter(),
 		grpcQueryRouter:  NewGRPCQueryRouter(),
```
Comment (3100) [inline app/baseapp.go:68]: Related to the above comment, is there a problem with baseapp.NewGRPCQueryRouter?

**Review by 3100 — APPROVED** [review]
@bluele 
Looks good, but I added some comments.

File: app/baseapp.go (hunk 1 above)
Comment (bluele) [reply app/baseapp.go:42]: This is because `GRPCQueryRouter.returnTypeOf` is private method.

File: app/baseapp.go (hunk 2 above)
Comment (bluele) [reply app/baseapp.go:68]: See above comment.

**Pull request merged**